speech.py

```diff
@@ -74,7 +78,33 @@ def _intialize_client():
     return boto3.Session(**aws_config).client("polly")
 
 
-# TODO: Add caching for identical responses
+def ttlcache(ttl=datetime.timedelta(seconds=600)):
```

Comment:
Looks good except that I see no code anywhere to cull expired data from the cache?

Reply:
Yeah, haha, in any case silly to roll our own. Replacing with TTLCache from the cachetools package.